metadata-ingestion/src/datahub/ingestion/source/snowflake/snowflake_config.py

```diff
@@ -163,26 +163,13 @@ class SnowflakeConfig(
         default=True,
         description="If enabled, populates the snowflake table-to-table and s3-to-snowflake table lineage. Requires appropriate grants given to the role and Snowflake Enterprise Edition or above.",
```

Comment:
should we update this `include_table_lineage` description here to emphasize that this enables view lineage too?

Reply:
The view upstream lineage would be ingested always irrespective of this flag.